release notes/v0.38.2.md

```diff
@@ -0,0 +1,94 @@
+k6 v0.38.2 is a patch release containing a couple of bugfixes!
+
+### Threshold over sub-metrics without samples would result in `NaN` ([#2520](https://github.com/grafana/k6/issues/2520))
+
+There was a bug in thresholds applied to sub-metrics set to `abortOnFail` that would lead to their result being `NaN` rather than a numerical value. The following script, for instance:
```

Comment:
I would argue the predominant problem was that we evaluate thresholds which will likely abort the test before they have any chance of actually passing such as `rate > 0.9` when no samples for the given metric have been emitted. 

The `NaN` was a specific case and it's mostly relevant as teh fix in v0.38.0 was supposed to make certain we will evaluate thresholds for which no metrics are emitted ... at least *once*. 

I would argue we should probably print a warning about such thresholds - once with no metrics as just because they pass doesn't mean that everything works correctly - maybe the user missconfigured something. But that is definitely an issue for v0.39.0 and beyond

Reply:
Fair point! I would still keep the emanation of the issue as being the `NaN` appearance as it might be the error that drives users to look for a solution, but I'll add the underlying reasoning then 👍🏻